packages/server/postgres/migrations/1619586448010_addEqCheckedFieldToUserAndTeam.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate';
+
+export const shorthands: ColumnDefinitions | undefined = undefined;
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.sql(`
+    ALTER TABLE "User"
+      ADD COLUMN "eqChecked" TIMESTAMPTZ NOT NULL DEFAULT '-infinity'::timestamptz;
+  
+    ALTER TABLE "Team"
+      ADD COLUMN "eqChecked" TIMESTAMPTZ NOT NULL DEFAULT '-infinity'::timestamptz;
+  
+    CREATE INDEX IF NOT EXISTS "idx_User_eqChecked" ON "User"("eqChecked");
+    CREATE INDEX IF NOT EXISTS "idx_Team_eqChecked" ON "User"("eqChecked");
```

Comment:
AH! looks like I missed this. (Should be "Team"?)
With the new plan of attack, let's just remove this migration & we can get by with a `COUNT`